src/msg/Message.cc

```diff
@@ -211,6 +211,10 @@
 #include "messages/MOSDPGUpdateLogMissing.h"
 #include "messages/MOSDPGUpdateLogMissingReply.h"
 
+#ifdef WITH_BLKIN
+#include "msg/Messenger.h"
```

Comment:
why? i don't see anything new in this .cc file.

Reply:
@tchaikov WITH_BLKIN = ON, compiled code needs to call method on messaner
`  const auto endpoint = msgr->get_trace_endpoint();
`